consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/BlockTimer.java

```diff
@@ -81,10 +81,10 @@ public synchronized boolean isRunning() {
    * Starts a timer for the supplied round cancelling any previously active block timer
    *
    * @param round The round identifier which this timer is tracking
-   * @param chainHeadHeader The header of the chain head
+   * @param getTimestamp The header of the chain head
    */
   public synchronized void startTimer(
-      final ConsensusRoundIdentifier round, final BlockHeader chainHeadHeader) {
+      final ConsensusRoundIdentifier round, final Supplier<Long> getTimestamp) {
```

Comment:
nit: `getTimeStamp` -> `currentTimeStamp` or `headerTimeStamp`?

Reply:
renamed to headerTimeStamp